### PR TITLE
Add a bunch of Gannett sites

### PR DESCRIPTION
**Pull request by stucka**
I built out site stubs for about 200 Gannett sites.

I'm less certain about the bundles. I drafted bundles for a bunch of U.S. states that didn't exist before. There are potential conflicts (e.g., illinois vs. chicago). I didn't add california as that was already fragmented. I created a new-york-state as there was already a new-york-city. Note many states have multiple time zones, per https://github.com/palewire/news-homepages/issues/283

### CODE REVIEW

File: newshomepages/sources/bundles.csv [hunk 1]
```diff
@@ -1,45 +1,67 @@
...
+oklahoma,Oklahoma,America/Chicago,Oklahoma City
+pennsylvania,Pennsylvania,America/New_York,Harrisburg
+rhode-island,Rhode Island,America/New_York,Providence
+south-carolina,South Carolina,America/New_York,Columbia
+south-dakota,South Dakota,America/Chicago,Pierre
+tennessee,Tennessee,America/Chicago,Nashville
+vermont,Vermont,America/New_York,Montpelier
+virginia,Virginia,America/New_York,Richmond
+west-virginia,West Virginia,America/New_York,Charleston
+,,,
```
Comment (palewire) [inline newshomepages/sources/bundles.csv:67]: @stucka can we cut this empty line?

**Comment by palewire** [comment]
Dude, this is incredible. Have you run screenshots or another collector on all the sites to verify there aren't parsing errors in the CSV?

**Comment by stucka** [comment]
CSV additions were done programmatically. I tested a sample earlier; will let it run overnight on the whole stretch. I tried to wait on the full run earlier today but there's just too much overhead.

**Comment by palewire** [comment]
Copy. I might add a new Gannett bundle to all of the sites and then test a run on that.

**Comment by stucka** [comment]
I'd be glad to make a Gannett bundle -- I didn't know if that was desirable or not.

FYI, just running the testing in series here it's hung up at least twice so far, both times trying to load the thanks-for-installing-AdGuard page.

**Comment by palewire** [comment]
Try running headless. Check the xvfb-run example in the actions.

**Comment by stucka** [comment]
There's a failure in test_sites() that may be from some hard coding; I don't know what that's about.

My testing last night (apart from ~five that stalled) was successful.

As far as adding sites, I'd like to do it once and get it right. How's this sound @palewire ? I can do all this, if desired.
-- Build a california bundle; add it back to the proposed Gannett California sites, and tag in existing bay-area bundle sites, such that bay-area acts as a subset of california.
-- Take proposed new-york-state bundle and also append to existing new-york-city sites, such that new-york-city generally acts as a subset of new-york-state.
-- Take proposed Gannett sites and tag with new gannett bundle. Identify existing Gannett sites and add in gannett bundle.
-- Check for stray blank lines at the end of things.
-- Optionally sort bundles.csv and sites.csv.

**Comment by palewire** [comment]
Thumbs up to all of those proposals. Let me know how I can help.

**Comment by stucka** [comment]
I think I'm there.

**Comment by palewire** [comment]
Great. I'll pull it down and look at it. Good prompt for me to fix the flake8 bugs in master while I'm at it.

**Comment by palewire** [comment]
The Cincinnati Enquirer was in there twice. I've removed the duplicate.

**Comment by stucka** [comment]
Nice catch on Cinci! The dangers of case-sensitive handles.

**Comment by palewire** [comment]
You're right about that. If you dig around the code you can see I'm often forcing them to lowercase internally when I create things like internetarchive slugs. Maybe we should force the spreadsheet lowercase as well.

**Comment by palewire** [comment]
Alrighty. Bombs away! This is our biggest addition to the index yet. I've added two more batches to our routine runs to try and ease the impact. Should be interesting to see how it goes.

**Pull request merged**